src/components/banner/banner.spec.ts

```diff
@@ -0,0 +1,250 @@
+import {
+  elementUpdated,
+  expect,
+  fixture,
+  html,
+  unsafeStatic,
+} from '@open-wc/testing';
+import { spy } from 'sinon';
+
+import { IgcBannerComponent, defineComponents } from '../../index.js';
```

Comment:
```suggestion
import { defineComponents } from '../common/definitions/defineComponents.js';
import IgcBannerComponent from './banner.js';
```

Nitpick but there is no need to import these from the top barrel index